test/jasmine/tests/scattermapbox_test.js

```diff
@@ -800,8 +800,8 @@ describe('@noCI Test plotly events on a scattermapbox plot:', function() {
             expect(pt.lon).toEqual(10, 'points[0].lon');
             expect(pt.pointNumber).toEqual(0, 'points[0].pointNumber');
 
-            expect(evt.clientX).toEqual(pointPos[0], 'event.clientX');
-            expect(evt.clientY).toEqual(pointPos[1], 'event.clientY');
+            expect(evt.clientX).toBeDefined('event.clientX');
+            expect(evt.clientY).toBeDefined('event.clientY');
```

Comment:
why did these have to change?

Reply:
Oops. My mistake. I shouldn't have committed this. Good :eye: Reverted in https://github.com/plotly/plotly.js/pull/2361/commits/0ef2ba68623097d39daa1f0cefd6afbc8b064860